hack/install/install_cni.sh

```diff
@@ -20,7 +22,14 @@ cni::install_cni() {
   workdir="${GOPATH}/src/${pkg}"
 
   # downloads github.com/containernetworking/plugins
-  go get -u -d "${pkg}"/...
+  if [ ! -d "${workdir}" ]; then
```

Comment:
If fix cni version, we can just download releases from https://github.com/containernetworking/plugins/releases/download/${CNI_VERSION}/cni-plugins-amd64-${CNI_VERSION}.tgz

Reply:
In fact, the size of the source code package is smaller than the size of the `.tgz`. In addition, we will be freedom to operate the branch . So no need to change it, IMO. WDYT? Anyway, thanks for your advice. :)